### DISCUSSION TITLE
Storybook fails on initialization

### DISCUSSION THREAD

**Discussion by JoshuaCS94** [storybookjs/storybook#13977] [Help]
I just created a fresh React app with `npx create-react-app` and can't get Storybook initialized by `npx sb init`. Maybe I'm missing something but I think that's what the installation guide says. I tried using the `--use-npm` option but keeps complaning about yarn not finding any version of those addons... do I have to install them by hand? I guess I shouldn't have to

![image](https://user-images.githubusercontent.com/23385700/108600378-fd634f80-7364-11eb-9d54-c1573904de73.png)

Thanks for the help.

**Comment by shilman** [comment]
Haven't seen this issue before. The following worked for me with `npm 7.5.4`:

```
npx --legacy-peer-deps sb init --use-npm
```

**Reply by JoshuaCS94** [reply]
Still not working, even with the `--legacy-peer-deps` option. I just searched for `"Unable to find versions"` and got 3 results, all from the same folder [js-package-manager](https://github.com/storybookjs/storybook/blob/4f5ab9fe9e590da7b841ec37cb1bed8d6327ea4b/lib/cli/src/js-package-manager). Not related to my problem but I was wondering why even the `--use-npm` options complained about not finding a version with yarn. The problem is [it logs `"...using yarn"` on error,](https://github.com/storybookjs/storybook/blob/4f5ab9fe9e590da7b841ec37cb1bed8d6327ea4b/lib/cli/src/js-package-manager/NPMProxy.ts#L50) is that a bug? :sweat_smile:
